oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/otlp/OpenTelemetryMetricRequestProcessor.java

```diff
@@ -334,4 +339,59 @@ private Stream<? extends Metric> adaptMetrics(
         }
         throw new UnsupportedOperationException("Unsupported type");
     }
+
+    public static String anyValueToString(AnyValue value) {
+        if (value.hasStringValue()) {
+            return value.getStringValue();
+        } else if (value.hasBoolValue()) {
+            return Boolean.toString(value.getBoolValue());
+        } else if (value.hasIntValue()) {
+            return Long.toString(value.getIntValue());
+        } else if (value.hasDoubleValue()) {
+            return Double.toString(value.getDoubleValue());
+        } else if (value.hasArrayValue()) {
+            return arrayValueToString(value.getArrayValue());
+        } else if (value.hasKvlistValue()) {
+            return keyValueListToString(value.getKvlistValue());
```

Comment:
Don't do this, I am not sure this is suitable for real use cases.

Reply:
Such as value could include comma, which would make a list of 4 looks like a list of 5 elements.